apps/finance/app/src/lib/token-utils.js

```diff
@@ -38,3 +43,24 @@ export const tokenDataFallback = (tokenAddress, fieldName, networkType) => {
   }
   return fallbacksForNetwork.get(addressWithoutChecksum)[fieldName] || null
 }
+
+export async function getTokenSymbol(app, address) {
+  let tokenAbi = [].concat(tokenBalanceOfAbi, tokenDecimalsAbi, tokenSymbolAbi)
```

Comment:
Technically we only need `[tokenSymbolAbi]` here and `[tokenSymbolBytesAbi]` later.

Reply:
So instead of 
`let tokenAbi = [].concat(tokenBalanceOfAbi, tokenDecimalsAbi, tokenSymbolAbi)`
this:
`let token = app.external(address, tokenSymbolAbi)` ?

Reply:
The API I believe still requires the ABI to be input as an array I think, so it'd be `let token = app.external(address, [tokenSymbolAbi])`

Reply:
mmm strange i tested it and it didn't  fail... But maybe it was something cached in my browser..